composer.json

```diff
@@ -21,7 +21,7 @@
   },
   "require": {
     "php": "^8.1",
-    "brick/math": "^0.12"
+    "brick/math": "^0.11 || ^0.12"
```

Comment:
maybe we should just keep everything as it was :see_no_evil:

Reply:
At least as long as RamseyUID will not update it's dependpencies